### PR TITLE
Fix startup command in dev

### PR DESCRIPTION
**Pull request by 10thfloor**
This PR fixes the startup command in development:

Context:
- We want to have a single command for starting the worker and the API server in dev, instead of having to start both processes manually. No need for multiprocess in dev.
- In 'production' we would like the ability to start both services in a separate process.

### CODE REVIEW

**Comment by psiemens** [comment]
It looks like this mixes in changes from the Heroku branch. Is the base branch wrong?

**Comment by kulame** [comment]
maybe you can follow the DevServices by quarkus.

https://quarkus.io/blog/quarkus-1-13-0-final-released/

it has a Zero Config Setup with DevServices solution.  it's cool.

**Comment by 10thfloor** [comment]
> It looks like this mixes in changes from the Heroku branch. Is the base branch wrong?

Correct, it is based on the Heroku branch, we'll need to merge that one first, then this one.
Or, we can merge this into the Heroku branch.

File: api/src/index.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import * as fcl from "@onflow/fcl";
 import yargs from "yargs/yargs";
 import { hideBin } from "yargs/helpers";
 
+import { Model } from "objection";
+
```
Comment (10thfloor) [inline api/src/index.ts:7]:
```suggestion
```

**Pull request merged**